src/main/java/com/github/rahulsom/genealogy/LastName.java

```diff
@@ -100,4 +146,22 @@ public Race getRace(double raceProbability) {
         return (Race) data[0][0];
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Why do we need this method instead of the lombok annotation `@EqualsAndHashCode`?

Reply:
Let's leave boring methods with warnings.